react/bulkImport/messages.ts

```diff
@@ -14,6 +14,19 @@ export type BulkImportMessageKey =
   | 'importReportTitle'
   | 'reportInformationTitle'
   | 'reportInformationDescription'
+  | 'reportScreenSuccessCount'
+  | 'reportScreenTitle'
+  | 'reportScreenDescription'
+  | 'reportScreenLabel'
+  | 'reportInformationInitialImport'
+  | 'reportInformationInitialUpload'
+  | 'reportInformationNextSteps'
+  | 'reportInformationStep1'
+  | 'reportInformationStep2'
+  | 'reportInformationStep3Upload'
+  | 'reportInformationStep3Import'
+  | 'reportInformationStep4'
+  | 'downloadReviewedLink'
```

Comment:
These message keys should have been in the document before, but I just realized now that they are not. They don't have any relation to the integrations.